deploy/kubernetes/base/controller-deployment.yaml

```diff
@@ -34,10 +34,15 @@ spec:
           image: public.ecr.aws/fsx-csi-driver/aws-fsx-csi-driver:v0.9.0
           imagePullPolicy: IfNotPresent
           args:
+            # - --mode={all,controller,node} # specify the driver mode
```

Comment:
Same idea as above, for my own understanding what is the default behavior when mode isn't specified? From what I can tell the switch statements are configured to throw an error. Why don't we specify the mode as controller here?